optuna/integration/allennlp.py

```diff
@@ -36,6 +33,8 @@ def dump_best_config(input_config_file: str, output_config_file: str, study: opt
             Note that :func:`~optuna.study.Study.optimize` must have been called.
 
     """
+    _jsonnet_imports.check()
```

Comment:
How about checking the availability of allennlp? It's also required, and the error message may be more helpful.

Reply:
Sorry I somehow missed that AllenNLP is required here. We probably only need the AllenNLP `_DeferredImportExceptionContextManager` .

Reply:
Fixed.